docs/docs/concepts/dialogue-understanding.mdx

```diff
@@ -21,9 +25,62 @@ In the CALM approach, _Dialogue Understanding_ aims to represent how the end use
 wants to progress the conversation.
 There is currently only one command generator available: the `LLMCommandGenerator`.
 
-## Using the LLMCommandGenerator in Your Bot
+Like its name indicates, the `CommandGenerator` generates "commands" that are then internally processed to
+trigger operations on the current conversation. Here is a reference of all supported commands:
```

Comment:
what about moving this further down the page? To me feels more natural to have explanations higher up and reference lower down.

Reply:
yes definitely

Reply:
0bb8817da2b